README.md

```diff
@@ -1296,6 +1296,45 @@ _You can enable the following settings in Xcode by running [this script](resourc
   
   </details>
 
+* <a id='extension-access-control'></a>(<a href='#extension-access-control'>link</a>) **Specify the access control for each declaration in an extension individually.** [![SwiftFormat: extensionAccessControl](https://img.shields.io/badge/SwiftFormat-anyObjectProtocol-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md##extensionaccesscontrol)
+
+  <details>
+
+  #### Why?
+
+  Specifying the access control on the declaration itself helps engineers more quickly determine the access control level of an individual declaration.
```

Comment:
Can we mention something about not having to look at the access control of the type you are extending? I think pointing out this locality issue would be great.

Reply:
What do you mean? Could you share an example?

Reply:
Yeah definitely. I meant encoding some of this in the guide itself.

<img width="949" alt="Screen Shot 2020-09-25 at 8 01 57 PM" src="https://user-images.githubusercontent.com/1791049/94328769-05f85b00-ff6a-11ea-9dd9-d081798132c4.png">

Reply:
At some point we may also want to go through and start saying "member declaration" for clarity, but I like that we're being consistent with the existing guide.

Reply:
Though I see we cover it below, so I actually think it's fine 👍 
<img width="704" alt="Screen Shot 2020-09-25 at 8 10 01 PM" src="https://user-images.githubusercontent.com/1791049/94328942-9e430f80-ff6b-11ea-9163-949f45f1d1e0.png">